src/components/calcite-action-pad/calcite-action-pad.scss

```diff
@@ -1,5 +1,5 @@
 :host {
-  display: flex;
+  display: inline-flex;
```

Comment:
The action pad was fillings its container. It seemed that behavior was *undesired* so I changed this to inline-flex to remedy that.